src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/AuthenticationHelper.cs

```diff
@@ -159,6 +171,10 @@ private static void SetBasicAuthToken(HttpRequestMessage request, NetworkCredent
             // Any errors in obtaining parameter return false and we don't proceed with auth
             if (string.IsNullOrEmpty(parameter))
             {
+                if (NetEventSource.IsEnabled)
+                {
+                    NetEventSource.AuthenticationInfo(authUri, $"Unable to find 'Digest' authentication token when authenticating with {(isProxyAuth ? "proxy" : "server")}");
```

Comment:
This line caused CI failure. No `authUri` in the context.

Reply:
Thanks -- I made a last minute change and should have checked it :)